{"totalCount":10,"grade":"D","repositories":[{"url":"https://github.com/MTES-MCT/sparte","vulnerabilityAlerts":{"totalCount":10,"nodes":[{"dismissedAt":null,"createdAt":"2023-08-29T23:52:40Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"jupyter-server"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-64x5-55rw-9974"},{"type":"CVE","value":"CVE-2023-40170"}],"references":[{"url":"https://github.com/jupyter-server/jupyter_server/security/advisories/GHSA-64x5-55rw-9974"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-40170"},{"url":"https://github.com/jupyter-server/jupyter_server/commit/87a4927272819f0b1cae1afa4c8c86ee2da002fd"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/jupyter-server/PYSEC-2023-157.yaml"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/XDKQAWQN6SQTOVACZNXYKEHWQXGG4DOF/"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NRP7DNZYVOIA4ZB3U3ZWKTFZEPYWNGCQ/"},{"url":"https://github.com/advisories/GHSA-64x5-55rw-9974"}]}}},{"dismissedAt":null,"createdAt":"2023-08-29T23:52:47Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"jupyter-server"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-r726-vmfq-j9j3"},{"type":"CVE","value":"CVE-2023-39968"}],"references":[{"url":"https://github.com/jupyter-server/jupyter_server/security/advisories/GHSA-r726-vmfq-j9j3"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-39968"},{"url":"https://github.com/jupyter-server/jupyter_server/commit/290362593b2ffb23c59f8114d76f77875de4b925"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/jupyter-server/PYSEC-2023-155.yaml"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/XDKQAWQN6SQTOVACZNXYKEHWQXGG4DOF/"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NRP7DNZYVOIA4ZB3U3ZWKTFZEPYWNGCQ/"},{"url":"https://github.com/advisories/GHSA-r726-vmfq-j9j3"}]}}},{"dismissedAt":null,"createdAt":"2023-12-05T18:26:07Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"jupyter-server"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-h56g-gq9v-vc8r"},{"type":"CVE","value":"CVE-2023-49080"}],"references":[{"url":"https://github.com/jupyter-server/jupyter_server/security/advisories/GHSA-h56g-gq9v-vc8r"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-49080"},{"url":"https://github.com/jupyter-server/jupyter_server/commit/0056c3aa52cbb28b263a7a609ae5f17618b36652"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/62LO7PPIAMLIDEKUOORXLHKLGA6QPL77/"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FG2JWZI5KPUYMDPS53AIFTZJWZD3IT6I/"},{"url":"https://github.com/advisories/GHSA-h56g-gq9v-vc8r"}]}}},{"dismissedAt":null,"createdAt":"2024-02-07T18:25:43Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-xxj9-f6rv-m3x4"},{"type":"CVE","value":"CVE-2024-24680"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-24680"},{"url":"https://docs.djangoproject.com/en/5.0/releases/security/"},{"url":"https://groups.google.com/forum/#%21forum/django-announce"},{"url":"https://www.djangoproject.com/weblog/2024/feb/06/security-releases/"},{"url":"https://github.com/django/django/commit/16a8fe18a3b81250f4fa57e3f93f0599dc4895bc"},{"url":"https://github.com/django/django/commit/55519d6cf8998fe4c8f5c8abffc2b10a7c3d14e9"},{"url":"https://github.com/django/django/commit/572ea07e84b38ea8de0551f4b4eda685d91d09d2"},{"url":"https://github.com/django/django/commit/c1171ffbd570db90ca206c30f8e2b9f691243820"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-28.yaml"},{"url":"https://github.com/advisories/GHSA-xxj9-f6rv-m3x4"}]}}},{"dismissedAt":null,"createdAt":"2024-02-09T10:48:44Z","securityVulnerability":{"severity":"HIGH","package":{"name":"cryptography"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-3ww4-gg4f-jr7f"},{"type":"CVE","value":"CVE-2023-50782"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-50782"},{"url":"https://access.redhat.com/security/cve/CVE-2023-50782"},{"url":"https://bugzilla.redhat.com/show_bug.cgi?id=2254432"},{"url":"https://github.com/pyca/cryptography/issues/9785"},{"url":"https://github.com/advisories/GHSA-3ww4-gg4f-jr7f"}]}}},{"dismissedAt":null,"createdAt":"2024-02-17T00:51:24Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"cryptography"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-9v9h-cgj8-h64p"},{"type":"CVE","value":"CVE-2024-0727"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-0727"},{"url":"https://github.com/openssl/openssl/commit/09df4395b5071217b76dc7d3d2e630eb8c5a79c2"},{"url":"https://github.com/openssl/openssl/commit/775acfdbd0c6af9ac855f34969cdab0c0c90844a"},{"url":"https://github.com/openssl/openssl/commit/d135eeab8a5dbf72b3da5240bab9ddb7678dbd2c"},{"url":"https://github.openssl.org/openssl/extended-releases/commit/03b3941d60c4bce58fab69a0c22377ab439bc0e8"},{"url":"https://github.openssl.org/openssl/extended-releases/commit/aebaa5883e31122b404e450732dc833dc9dee539"},{"url":"https://www.openssl.org/news/secadv/20240125.txt"},{"url":"https://github.com/github/advisory-database/pull/3472"},{"url":"https://github.com/openssl/openssl/pull/23362"},{"url":"https://github.com/alexcrichton/openssl-src-rs/commit/add20f73b6b42be7451af2e1044d4e0e778992b2"},{"url":"https://github.com/pyca/cryptography/commit/3519591d255d4506fbcd0d04037d45271903c64d"},{"url":"https://security.netapp.com/advisory/ntap-20240208-0006"},{"url":"https://github.com/advisories/GHSA-9v9h-cgj8-h64p"}]}}},{"dismissedAt":null,"createdAt":"2024-02-21T20:19:11Z","securityVulnerability":{"severity":"HIGH","package":{"name":"cryptography"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-6vqw-3v5j-54x4"},{"type":"CVE","value":"CVE-2024-26130"}],"references":[{"url":"https://github.com/pyca/cryptography/security/advisories/GHSA-6vqw-3v5j-54x4"},{"url":"https://github.com/pyca/cryptography/pull/10423"},{"url":"https://github.com/pyca/cryptography/commit/97d231672763cdb5959a3b191e692a362f1b9e55"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-26130"},{"url":"https://github.com/advisories/GHSA-6vqw-3v5j-54x4"}]}}},{"dismissedAt":null,"createdAt":"2024-03-18T21:31:58Z","securityVulnerability":{"severity":"LOW","package":{"name":"django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-vm8q-m57g-pff3"},{"type":"CVE","value":"CVE-2024-27351"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-27351"},{"url":"https://docs.djangoproject.com/en/5.0/releases/security"},{"url":"https://groups.google.com/forum/#%21forum/django-announce"},{"url":"https://www.djangoproject.com/weblog/2024/mar/04/security-releases"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-47.yaml"},{"url":"https://github.com/advisories/GHSA-vm8q-m57g-pff3"}]}}},{"dismissedAt":null,"createdAt":"2024-03-20T18:09:41Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"black"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-fj7x-q9j7-g6q6"},{"type":"CVE","value":"CVE-2024-21503"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-21503"},{"url":"https://github.com/psf/black/commit/f00093672628d212b8965a8993cee8bedf5fe9b8"},{"url":"https://github.com/psf/black/releases/tag/24.3.0"},{"url":"https://security.snyk.io/vuln/SNYK-PYTHON-BLACK-6256273"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/black/PYSEC-2024-48.yaml"},{"url":"https://github.com/advisories/GHSA-fj7x-q9j7-g6q6"}]}}},{"dismissedAt":null,"createdAt":"2024-04-03T16:12:31Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"pillow"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-44wm-f244-xhp3"},{"type":"CVE","value":"CVE-2024-28219"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-28219"},{"url":"https://pillow.readthedocs.io/en/stable/releasenotes/10.3.0.html#security"},{"url":"https://github.com/python-pillow/Pillow/commit/2a93aba5cfcf6e241ab4f9392c13e3b74032c061"},{"url":"https://github.com/advisories/GHSA-44wm-f244-xhp3"}]}}}]},"grade":"D"}]}